src/e2e-test/java/io/cdap/plugin/datastore/stepsdesign/DataStoreLocators.java

```diff
@@ -0,0 +1,13 @@
+package io.cdap.plugin.datastore.stepsdesign;
```

Comment:
Please add the license and java doc. Refer any existing action class.

Reply:
added.